spec/operators/combineAll-spec.js

```diff
@@ -4,6 +4,416 @@ var Observable = Rx.Observable;
+  });
+
+  it('should work with throw and never', function () {
+    var e1 =    hot('---^----#', null, 'wokka wokka');
+    var e1subs =       '^    !';
+    var e2 =    hot('---^-----------');
+    var e2subs =       '^    !';
+    var expected =     '-----#';
+
+    var result = Observable.of(e1, e2).combineAll(function (x, y) { return x + y; });
+
+    expectObservable(result).toBe(expected, null, 'wokka wokka');
+    expectSubscriptions(e1.subscriptions).toBe(e1subs);
+    expectSubscriptions(e2.subscriptions).toBe(e2subs);
+  });
+
+  it('should work with some and throw', function () {
+    var e1 =    hot('---^----a---b--|', { a: 1, b: 2 });
+    var e1subs =       '^  !';
+    var e2 =    hot('---^--#', null, 'wokka wokka');
+    var e2subs =       '^  !';
+    var expected =     '---#';
+
+    var result = Observable.of(e1, e2).combineAll(function (x, y) { return x + y; });
+
+    expectObservable(result).toBe(expected, { a: 1, b: 2}, 'wokka wokka');
+    expectSubscriptions(e1.subscriptions).toBe(e1subs);
+    expectSubscriptions(e2.subscriptions).toBe(e2subs);
+  });
+
+  it('should work with throw and some', function () {
+    var e1 =    hot('---^--#', null, 'wokka wokka');
+    var e1subs =       '^  !';
+    var e2 =    hot('---^----a---b--|', { a: 1, b: 2 });
+    var e2subs =       '^  !';
+    var expected =     '---#';
+
+    var result = Observable.of(e1, e2).combineAll(function (x, y) { return x + y; });
+
+    expectObservable(result).toBe(expected, { a: 1, b: 2}, 'wokka wokka');
+    expectSubscriptions(e1.subscriptions).toBe(e1subs);
+    expectSubscriptions(e2.subscriptions).toBe(e2subs);
+  });
+
+  it('should handle throw after complete left', function () {
+    var left =  hot('--a--^--b---|', { a: 1, b: 2 });
+    var leftSubs =       '^      !';
+    var right = hot('-----^--------#', null, 'bad things');
+    var rightSubs =      '^        !';
+    var expected =       '---------#';
+
+    var result = Observable.of(left, right).combineAll(function (x, y) { return x + y; });
+
+    expectObservable(result).toBe(expected, null, 'bad things');
+    expectSubscriptions(left.subscriptions).toBe(leftSubs);
+    expectSubscriptions(right.subscriptions).toBe(rightSubs);
+  });
+
+  it('should handle throw after complete right', function () {
+    var left =   hot('-----^--------#', null, 'bad things');
+    var leftSubs =        '^        !';
+    var right =  hot('--a--^--b---|', { a: 1, b: 2 });
+    var rightSubs =       '^      !';
+    var expected =        '---------#';
+
+    var result = Observable.of(left, right).combineAll(function (x, y) { return x + y; });
+
+    expectObservable(result).toBe(expected, null, 'bad things');
+    expectSubscriptions(left.subscriptions).toBe(leftSubs);
+    expectSubscriptions(right.subscriptions).toBe(rightSubs);
+  });
+
+  it('should handle interleaved with tail', function () {
+    var e1 = hot('-a--^--b---c---|', { a: 'a', b: 'b', c: 'c' });
```

Comment:
Good catch. Since these tests are from combineLatest, I'll have to update those to.